deploy/vagrant/create

```diff
@@ -176,17 +176,18 @@ fi
 masters=($(echo $masters | tr '\n' ' ' | tr -d '\r'))
 
 if [[ "$TACHYON_MASTERS" -gt 1 ]]; then
-  info ">>> Tachyon is in Fault Tolerant mode, all Tachyon masters: <<<"
+  info "Tachyon is in Fault Tolerant mode, all Tachyon masters:"
+  for master in ${masters[@]}; do
+    info "Tachyon master: ${master}"
+    info "Trying to curl Tachyon web UI at ${master}:19999"
+    curl "${master}:19999" >/dev/null
+    if [[ "$?" == "0" ]]; then
+      info "Master public IP for Tachyon is ${master}"
```

Comment:
Does this line print for only the leader master, or all masters? If it is the leader master, it should print out the full ip:19999 in the output somewhere.